test/ex_sdp/attribute/rtcp_feedback_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule ExSDP.Attribute.RTCPFeedbackTest do
+  use ExUnit.Case, async: true
+
+  alias ExSDP.Attribute.RTCPFeedback
+
+  test "Basic feedback types parsing" do
```

Comment:
I'm not a big fan of such verbosity in test names. The fact it's `RTCPFeedback` module is already there in the failure message as it includes the test module name. Especially when the test exclusively consists of `RTCPFeedback.parse` calls.
```
1) test Basic feedback types parsing (ExSDP.Attribute.RTCPFeedbackTest)

     Assertion with == failed
     code:  assert RTCPFeedback.parse("98 goog-remb") == {:ok, %RTCPFeedback{pt: 99, feedback_type: :remb}}
```

This is more or less the convention that the Elixir itself follows - the names are often even more concise:
https://github.com/elixir-lang/elixir/blob/main/lib/elixir/test/elixir/enum_test.exs

Following that here it would be:
```suggestion
  describe "parse/1" do
    test "basic feedback types" do
    ..
    test "unknown feedback type" do
```

so my approach is somewhere in between. To sum up, I think the module name should not be included in the test name (this is already covered by test module naming), the rest is quite subjective and I'm open to discussion, but IMO  the simpler the test case the simpler the name should be. I guess we should work out some convention for the whole team

Reply:
I see, but I believe that all other tests are written in the convention `test (if) x handles y`. Introducing the new style means that we want to gradually rewrite all other tests? 

Now, if someone other contributes to this repo, it won't know which scheme to use